### PR TITLE
OXT-1026 : Configure Xenstored logging

### PR DESCRIPTION
**Pull request by dozylynx**
Update OCaml Xenstored config file log settings.
Grant SELinux permissions required by the daemon.

Signed-off-by: Christopher Clark <christopher.clark6@baesystems.com>

### CODE REVIEW

File: recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff [hunk 1]
```diff
@@ -454,17 +454,19 @@ Index: refpolicy/policy/modules/contrib/xen.te
...
-@@ -447,6 +550,9 @@ kernel_read_xen_state(xenstored_t)
+@@ -447,6 +550,11 @@ kernel_read_xen_state(xenstored_t)
  dev_filetrans_xen(xenstored_t)
  dev_rw_xen(xenstored_t)
  dev_read_sysfs(xenstored_t)
 +dev_create_generic_dirs(xenstored_t)
 +dev_manage_xen(xenstored_t)
 +corecmd_search_bin(xenstored_t)
++kernel_read_fs_sysctls(xenstored_t)
++virt_read_config(xenstored_t)
```
Comment (jean-edouard) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: Why is this needed? Don't xenstored config files have their own type?

Comment (jean-edouard) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: You should just need `virt_search_dirs(xenstored_t)` to enter /etc/xen

Comment (dozylynx) [inline recipes-security/refpolicy/refpolicy-mcs-2.20141203/patches/policy.modules.contrib.xen.diff]: `virt_search_dirs(xenstored_t)` looks good. PR refreshed.

**Comment by jean-edouard** [comment]
LGTM, smoke tested, merging soon.
